### ISSUE TITLE
call `render` instead of `show` in accessible views

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#214147]
`render` was introduced so that listeners aren't recreated, this should be adopted elsewhere.

https://github.com/microsoft/vscode/blob/660018a75f944cd9cc27073eef4fc6918ef7ae23/src/vs/workbench/contrib/accessibility/browser/accessibleView.ts#L487
